js/renderer/texturesprite_renderer.js

```diff
@@ -59,7 +66,31 @@ LayerScope.SpriteView = {
     this._resizeSprites(this._textures, this._colors);
   },
 
-  _createTexSprites: function SV_createTexSprites(frame, $panel) {
+  _sortSpriteByLayerID: function SV_sortSpriteByLayerID(layerID, $sprite) {
+    if (this._sortedLayerIDs === [] || this._sortedSprites === []) {
+      this._sortedLayerIDs.push(layerID);
+      this._sortedSprites.push($sprite);
+      return;
+    }
+
+    var index = -1;
+    for (i = 0; i < this._sortedLayerIDs.length; i++) {
+      if (layerID < this._sortedLayerIDs[i]) {
+        index = i;
+        break;
```

Comment:
this._sortedLayerIDs.splice(index, 0, layerID);
  this._sortedSprites.splice(index, 0, $sprite);
  return;